scripts/deploy.sh

```diff
@@ -78,6 +78,13 @@ else
   echo "CLUSTER_REGION is set to ${CLUSTER_REGION}"
 fi
 
+if [ -z "${ENABLE_PRIVATE_CLUSTER}" ]; then
```

Comment:
We probably ought to trap non boolean values near here.

Reply:
We should still do this

Reply:
I'm wondering if this is a "do it nowwww" thing? as the instructions in env are pretty clear about it being boolean.
I'm on the fence about the "when" of this one.